samples/callbacks/Callbacks_3/WebSender/Startup.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.DependencyInjection;
+
+public class Startup
```

Comment:
More as a general question, feel free to ignore:

Is using the startup class still a thing or aren't most people moving towards using the `ConfigureXYZ` overloads on the builder in the program main?

Reply:
I kept it consistent with the other samples to make it easier to update them all at once:

- https://github.com/Particular/docs.particular.net/blob/0874d4df530bd71014e8e8d79ede7036f5f772f5/samples/dependency-injection/aspnetcore/Core_7.2/Sample/Program.cs#L33
- https://github.com/Particular/docs.particular.net/blob/0874d4df530bd71014e8e8d79ede7036f5f772f5/samples/web/send-from-aspnetcore-webapi/Core_7.2/WebApplication/Program.cs#L23
- https://github.com/Particular/docs.particular.net/blob/ed8459f17ec30a56309b8a16c9fb5beaf792c363/samples/web/send-from-aspnetcore-webapi/Core_8/WebApplication/Program.cs#L23
- etc...